internal/olm/operator/registry/index/registry_pod.go

```diff
@@ -77,8 +77,11 @@ type RegistryPod struct { //nolint:maligned
 	// The secret's key for this file must be "cert.pem".
 	CASecretName string
 
-	// SkipTLS controls wether to ignore SSL errors while pulling bundle image from registry server.
-	SkipTLS bool `json:"SkipTLS"`
+	// skipTLSVerify controls wether to ignore SSL errors while pulling bundle image from registry server.
+	SkipTLSVerify bool `json:"SkipTLSVerify"`
+
+	// skipTLSVerify controls wether to ignore SSL errors while pulling bundle image from registry server.
```

Comment:
Copy/paste typo here

Reply:
Yes, updated the description for UseHTTP variable.